test/suite/test_s3_store01.py

```diff
@@ -56,14 +57,21 @@ def test_local_basic(self):
        
         # Test flush functionality and flushing to cache and checking if file exists.
         f = open(filename, 'wb')
-        outbytes = ('Ruby\n'*100).encode()
+        outbytes = ('MORE THAN ENOUGH DATA\n'*100000).encode()
         f.write(outbytes)
         f.close()
 
         s3_store.ss_flush(session, fs, filename, object_name)
         s3_store.ss_flush_finish(session, fs, filename, object_name)
         self.assertTrue(fs.fs_exist(session, filename))
 
+        # fh = fs.fs_open_file(session, filename, FileSystem.open_file_type_data, FileSystem.open_create)
```

Comment:
why this commented line is here?

Reply:
I wanted to test the case where a file is opened in non-readonly mode; as`S3Open` should fail in this case. However currently the python test is hanging whenever a storage source interface function returns with a non-zero exit code. I'm not sure if fixing this in the scope of this ticket; I will remove it.